packages/terra-table/src/proptypes/subsectionShape.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+import rowShape from './rowShape';
+
+const sectionShape = PropTypes.shape({
```

Comment:
A [documentation page](https://github.com/cerner/terra-framework/blob/cc25304e3fcaa751253ae37f6763f763566b1b44/packages/terra-framework-docs/src/terra-dev-site/doc/table/About.1.doc.mdx?plain=1#L155) needs to be updated with section shape info similar to the way we have for section

Reply:
Added here: 2395c42a838cd07a92ab4f7a69f793eea5b14996